helm_deploy/values-dev.yaml

```diff
@@ -11,6 +11,7 @@ generic-service:
 
   env:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
+    GOV_UK_API_URL: "https://www.gov.uk"
```

Comment:
as this doesn't change you could put it in `values.yml` rather than in each env file

Reply:
Added as env variable in `values.yml`